internal/engine/postgresql/catalog.go

```diff
@@ -5,7 +5,7 @@ import "github.com/kyleconroy/sqlc/internal/sql/catalog"
 func NewCatalog() *catalog.Catalog {
 	c := catalog.New("public")
 	c.Schemas = append(c.Schemas, pgTemp())
-	c.Schemas = append(c.Schemas, pgCatalog())
+	c.Schemas = append(c.Schemas, genPGCatalog())
```

Comment:
This is a bit scary! I'm curious if we'll have to walk this specific change back.